catalog/plugin/catalog-plugin-configuration-mapping/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+        xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <reference id="expansionService" interface="ddf.security.expansion.Expansion"
+               filter="(mapping=security.metacard.attribute.mapping)"/>
+
+    <bean id="configurationMappingPlugin"
+          class="org.codice.ddf.catalog.plugin.configuration.mapping.ConfigurationMappingPlugin">
+        <property name="expansionService" ref="expansionService"/>
```

Comment:
Documenting the expected input format should be sufficient, we don't extensively validate that in other configurable areas.

A downside of using the expansion service is that those values are used elsewhere as well. It may not always be desirable or intended to declare a value as acceptable just so that it can be rewritten into the value that is actually desired. If the expansion service's internal logic has value, perhaps it should be refactored into a library that both it and this plugin can use?